src/Infrastructure/Model/Shopware6Product.php

```diff
@@ -646,12 +669,12 @@ public function jsonSerialize(): array
         if (null !== $this->options) {
             $data['options'] = $this->options;
         }
-        if (null !== $this->media) {
+        if (false === empty($this->media)) {
             foreach ($this->media as $media) {
                 $data['media'][] = $media->jsonSerialize();
             }
         }
-        if (null !== $this->configuratorSettings) {
+        if (false === empty($this->configuratorSettings)) {
```

Comment:
ten if i 2 powyzej nie sa potrzebne bo gdy tablica jest pusta to petla sie nie wykona

Reply:
ok - słusznie